packages/cosmic-swingset/src/decimals.js

```diff
@@ -0,0 +1,35 @@
+import { assert, details as X } from '@agoric/assert';
+
+export const decimalStrToFraction = decimalStr => {
+  const match = decimalStr.match(/^(\d+)(\.(\d*))?$/);
+  assert(match, X`${decimalStr} is not a decimal string`);
+  const decimals = match[3] || '';
+  const numeratorStr = `${match[1]}${decimals}`;
+  const numerator = BigInt(numeratorStr);
+  const denominator = 10n ** BigInt(decimals.length);
+  return [numerator, denominator];
+};
+
+export const fractionToDecimalStr = (numerator, denominator) => {
+  assert.typeof(numerator, 'bigint');
+  assert.typeof(denominator, 'bigint');
+  let decimalPlaces = 0;
+  while (denominator > 1n) {
+    assert.equal(denominator % 10n, 0n, X`${denominator} is not a power of 10`);
```

Comment:
> Are you sure you want to store the denominator rather than the exponent

Yes, I'd rather this library grew up into full rationals rather than hardcoding the `10**n` dependency in anything except for the places where it hasn't been completed.

Reply:
Withdrawn.  Now we just use BigInt beans instead of fractions.